homeassistant/components/ads/__init__.py

```diff
@@ -37,6 +56,10 @@
 CONF_ADS_VAR = "adsvar"
 CONF_ADS_VAR_BRIGHTNESS = "adsvar_brightness"
 CONF_ADS_VAR_POSITION = "adsvar_position"
+CONF_ADS_VAR_SET_POS = "adsvar_set_position"
+CONF_ADS_VAR_OPEN = "adsvar_open"
+CONF_ADS_VAR_CLOSE = "adsvar_close"
+CONF_ADS_VAR_STOP = "adsvar_stop"
```

Comment:
I suggest that as preliminary work (in another PR) you move all these constants out of `init.py` into a new `const.py` file.

Reply:
If you prefer, we can do this inside #63390, or you can create your own PR.
Once done, it will make this PR smaller and therefore easier to review.

Reply:
Could we do this in a follow up?

Reply:
Better as preliminary...

Reply:
This PR is from Oct. I won't do another rebase...

Reply:
Chicken and egg. Maybe the PR was not getting attention because it is bigger than it needs to be.